rust/xaynet-server/src/settings/mod.rs

```diff
@@ -84,7 +86,7 @@ pub struct PetSettings {
     /// be greater or equal to `1` (i.e. `min_sum_count >= 1`), otherwise the PET protocol will be
     /// broken.
     ///
-    /// This parameter should only be used to enforce security constraints.
+    /// This parameter should only be used to enforce security constraints. Defaults to 1.
```

Comment:
mhh, i see what you mean. that was more related to the reference config file, but that's not really helpful in the end. i will remove the "defaults to" comments, since it effectively has no defaults.

we can think about adding defaults for everything later on in a separate story if needed.

Reply:
we have a `Default` impl for the struct, but it is only used in tests currently. i will move that behind a `#[cfg(test)]` for now. if we want to provide a proper impl of default values we should mark the respective fields as `Option<T>` and then replace `None` values with `Some(default)` values after validation.